src/util/dump.ts

```diff
@@ -42,7 +43,10 @@ export async function getAndProcessStylesheets(downloader: Downloader, links: Ar
                     if (!url.match('^data')) {
                         const filePathname = urlParser.parse(url, false, true).pathname;
                         if (filePathname) {
-                            const filename = pathParser.basename(filePathname).replace(/-.*x./, '.');
+                            const filename = pathParser.basename(filePathname).replace(/-.*x./, '.')
+                            + shouldConvertImageFilenameToWebp(pathParser.basename(filePathname).replace(/-.*x./, '.'), downloader.webp)
```

Comment:
Do that in two steps, to avoid calling `pathParser.basename(filePathname).replace(/-.*x./, '.')` two times.